spec/views/posts/show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'User post', type: :feature do
+  before(:example) do
+    @user = User.create(name: 'Elon Musk', photo: 'link/goes/here', bio: 'this is my bio')
+    @post_one = Post.create(title: 'My first post', text: 'Body text of the post', author_id: @user.id)
+    Post.create(title: 'My second post', text: 'Body text of the post', author_id: @user.id)
+    Post.create(title: 'My third post', text: 'Body text of the post', author_id: @user.id)
+    Post.create(title: 'My fourth post', text: 'Body text of the post', author_id: @user.id)
+    Comment.create(text: 'My first awesome comment straight from Kenya', author_id: @user.id, post_id: @post_one.id)
+
+    visit user_post_path(@user, @post_one.id)
+  end
+
+  it 'renders who wrote the post. ' do
+    expect(page.html).to include(@user.name)
+  end
+
+  it 'renders number of comments ' do
+    expect(page).to have_content(1)
+  end
+
+  it 'renders number of likes ' do
+    expect(page).to have_content(0)
+  end
+
+  it 'renders post body ' do
+    expect(page).to have_content(@post_one.text)
+  end
+
+  it 'renders username of each commentor ' do
+    expect(page.html).to include(@user.name)
+  end
+
+  it 'renders comment of each commentor ' do
+    expect(page).to have_content('My first awesome comment straight from Kenya')
+  end
```

Comment:
- [ ] @team great work on your test however as the previous reviewer requested kindly add a test that shows you can see the post's title. 
![image](https://user-images.githubusercontent.com/61408860/203827640-c1df7847-490d-45b7-98ae-a2a4b78245b6.png)

Reply:
- [ ] @team I also noted some tests are not passing when I run the test, kindly ensure all the tests pass.